webpack/assets/javascripts/services/ChartService.js

```diff
@@ -52,6 +58,7 @@ const getChartConfig = ({
     ...chartConfigForType,
     id,
     data: {
+      ...chartConfigForType.data,
```

Comment:
I think if we'll remove `{type: bar}` from the config object, then we can remove this line.